### DISCUSSION TITLE
How to configure this with `folke/lua-dev`?

### DISCUSSION THREAD

**Discussion by macintacos** [VonHeikemen/lsp-zero.nvim#41] [Q&A] [answered]
I'm having a bit of trouble getting this configuration set up properly with [`lua-dev`](https://github.com/folke/lua-dev.nvim). Can I get some pointers on how to make that work? Thanks!

**Comment by VonHeikemen** [comment, marked as answer]
Use `.configure()` to setup `sumneko_lua` with the options provided by lua-dev.

```lua
local lsp = require('lsp-zero')
lsp.preset('recommended')

local luadev = require('lua-dev').setup({
  -- add any options here, or leave empty to use the default settings
})

lsp.configure('sumneko_lua', luadev)

lsp.setup()
```

**Reply by macintacos** [reply]
Jeez, I was making it way more complicated than it apparently needed to be! :)

Thanks so much, this works perfectly. Appreciate the quick response!
